apps/storybook-react-native/.storybook/config.js

```diff
@@ -16,7 +12,7 @@ addons.setConfig({
 });
 
 addDecorator((Story) => (
-  <ThemeProvider colorSet={ColorSet.Brand} theme={Theme.Default}>
+  <ThemeProvider theme={Theme.Default}>
```

Comment:
Removing ColorSet instance and checking none remain in codebase

<img width="242" alt="Screenshot 2025-06-04 at 11 55 20 AM" src="https://github.com/user-attachments/assets/ce23400c-bd2f-42f1-b2d2-f51f9074387d" />